Makefile

```diff
@@ -18,6 +18,7 @@ docs/repro.%: docs/repro.%.txt docs/asciidoc.conf
 
 repro: repro.in
 	m4 -DREPRO_CONFIG_DIR=$(CONFDIR)/$(PROGNM) $< >$@
+	m4 -DTAG=$(TAG) $< >$@
```

Comment:
I think you need to add this to the previous command, otherwise you're overwriting the replaced `REPRO_CONFIG_DIR` again.

I don't know much about m4, but if it really does search+replace then `TAG` might be too generic. :)

Reply:
I think `REPRO_VERSION` would be better :)